adobe/spectrum-web-components
# Picker button

## Description
Add the Picker Button pattern
- include basic functional test
- add expansive VRTs

## Related issue(s)

- fixes #2558
- refs #478

## Motivation and context
Prep for Combobox delivery

## How has this been tested?
    1. Go [here](https://picker-button--spectrum-web-components.netlify.app/components/picker-button/)
    2. See that it's well documented
    1. Go [here](https://picker-button--spectrum-web-components.netlify.app/storybook/index.html?path=/story/picker-button-sizes--s)
    2. See that its visual delivery is well covered\

## Types of changes

## Checklist
